components/outputStates.js

```diff
@@ -0,0 +1,23 @@
+// outportStates.js
+
+var _ = require('underscore');
+var Lm = require('./Lm');
+
+var outportStates = [];
+
+// Returns the vni associated with the specified vnid.
+module.exports = function( outportName, newState ) { 
+ 
+   if ( arguments.length === 1 ) { 
+       if ( _.isUndefined( outportStates[outportName] ) ) { 
+           return;
+       } 
```

Comment:
Can lines 12-14 be deleted, since line 15 will return undefined anyway if outportStates[outportName] is undefined?